app/lib/active_storage/previewer/csv_previewer.rb

```diff
@@ -0,0 +1,47 @@
+require "csv"
+require "chunky_png"
```

Comment:
There's a chance Rails will auto-require these gems and we don't need to explicitly do it here